onto_tool/ontograph.py

```diff
@@ -646,18 +652,22 @@ def __create_class_count_query(self, limit):
               {
                 select (group_concat(?o;separator=' ') as ?src) where {
                   $pattern
-                  FILTER(!ISBLANK(?s))
+                  $bnode_filter
                   FILTER (!STRSTARTS(STR(?o), 'http://www.w3.org/2002/07/owl#'))
                 } group by ?s LIMIT $limit
               }
             } group by ?src
             """
         query_text = Template(class_query).substitute(
             pattern=self.__filtered_graph_pattern(str(RDF.type)),
+            bnode_filter=bnode_filter,
             limit=limit)
         return query_text
 
     def __create_predicate_query(self, predicate, predicate_type, limit):
+        bnode_filter = "FILTER(!ISBLANK(?s))" if not self.show_bnode_subjects else ""
+        bnode_and_literal_filter = "FILTER(!ISBLANK(?s) && ISLITERAL(?o))" if not self.show_bnode_subjects \
```

Comment:
I would break up the filter in the query to
```
     FILTER(ISLITERAL(?o))
     $bnode_filter
```
So you don't have to have two different substitutions, and it's a little clearer.